### PR TITLE
Specify tokenAddress to enable multi token transfer

### PR DESCRIPTION
**Pull request by frankli-dev**
## Related Issue Or Context

Closes: #<issue>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: bindings/ERC20Safe/ERC20Safe.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ var (
...
...0fd5b506004361061002b5760003560e01c806395601f0914610030575b600080fd5b6100666004803603606081101561004657600080fd5b506001600160a01b03813581169160208101359091169060400135610068565b005b826100758184308561007b565b50505050565b604080516001600160a01b0380861660248301528416604482015260648082018490528251808303909101815260849091019091526020810180516001600160e01b03166323b872dd60e01b17905261007590859060006060836001600160a01b0316836040518082805190602001908083835b6020831061010e5780518252601f1990920191602091820191016100ef565b6001836020036101000a0380198251168184511680821785525050505050509050019150506000604051808303816000865af19150503d8060008114610170576040519150601f19603f3d011682016040523d82523d6000602084013e610175565b606091505b5091509150816101c1576040805162461bcd60e51b8152602060048201526012602482015271115490cc8c0e8818d85b1b0819985a5b195960721b604482015290519081900360640190fd5b805115610075578080602001905160208110156101dd57600080fd5b5051610075576040805162461bcd60e51b815260206004820181905260248201527f45524332303a206f7065726174696f6e20646964206e6f742073756363656564604482015290519081900360640190fdfea264697066735822122086598f653c169c44dc775664949479f398ce7c800ae22f59a1b3175a3ec2a52864736f6c634300060c0033"
```
Comment (KaiserKarel) [inline bindings/ERC20Safe/ERC20Safe.go:33]: How was this derived? We need some way to check that this erc20 bin does not contain a backdoor.

File: bindings/ERC721/ERC721.go [hunk 2]
```diff
@@ -27,19 +27,19 @@ var (
...
...57261626c655365743a20696e646578206f7574206f6620626f756e64734552433732313a207472616e7366657220746f206e6f6e20455243373231526563656976657220696d706c656d656e7465724552433732313a207472616e7366657220746f20746865207a65726f20616464726573734552433732313a206f70657261746f7220717565727920666f72206e6f6e6578697374656e7420746f6b656e4552433732313a20617070726f76652063616c6c6572206973206e6f74206f776e6572206e6f7220617070726f76656420666f7220616c6c4552433732313a2062616c616e636520717565727920666f7220746865207a65726f20616464726573734552433732313a206f776e657220717565727920666f72206e6f6e6578697374656e7420746f6b656e456e756d657261626c654d61703a20696e646578206f7574206f6620626f756e64734552433732313a20617070726f76656420717565727920666f72206e6f6e6578697374656e7420746f6b656e4552433732313a207472616e73666572206f6620746f6b656e2074686174206973206e6f74206f776e4552433732314d657461646174613a2055524920717565727920666f72206e6f6e6578697374656e7420746f6b656e4552433732313a20617070726f76616c20746f2063757272656e74206f776e65724552433732313a207472616e736665722063616c6c6572206973206e6f74206f776e6572206e6f7220617070726f766564a264697066735822122089341c260a9643a1a12b051272c90a60af9c8caa6742e55036b08114d6b86f9564736f6c634300060c0033"
```
Comment (KaiserKarel) [inline bindings/ERC721/ERC721.go:33]: Same here; how was this derived?

File: bindings/ERC721MinterBurnerPauser/ERC721MinterBurnerPauser.go [hunk 3]
```diff
@@ -27,10 +27,10 @@ var (
...
...2206e6f7220617070726f76656420666f7220616c6c4552433732313a2062616c616e636520717565727920666f7220746865207a65726f20616464726573734552433732313a206f776e657220717565727920666f72206e6f6e6578697374656e7420746f6b656e456e756d657261626c654d61703a20696e646578206f7574206f6620626f756e64734552433732313a20617070726f76656420717565727920666f72206e6f6e6578697374656e7420746f6b656e4552433732314d657461646174613a2055524920736574206f66206e6f6e6578697374656e7420746f6b656e4552433732313a207472616e73666572206f6620746f6b656e2074686174206973206e6f74206f776e4552433732314d657461646174613a2055524920717565727920666f72206e6f6e6578697374656e7420746f6b656e4552433732313a20617070726f76616c20746f2063757272656e74206f776e65724552433732313a207472616e736665722063616c6c6572206973206e6f74206f776e6572206e6f7220617070726f7665644552433732314275726e61626c653a2063616c6c6572206973206e6f74206f776e6572206e6f7220617070726f766564416363657373436f6e74726f6c3a2063616e206f6e6c792072656e6f756e636520726f6c657320666f722073656c664552433732314d696e7465724275726e65725061757365723a206d7573742068617665206d696e74657220726f6c6520746f206d696e74a2646970667358221220aba99c9548a19b435813e549a48990d92fcfd1fb69b8a6d1ad5233b1a813c4e064736f6c634300060c0033"
```
Comment (KaiserKarel) [inline bindings/ERC721MinterBurnerPauser/ERC721MinterBurnerPauser.go:33]: How was this derived

File: bindings/ERC721Safe/ERC721Safe.go [hunk 4]
```diff
@@ -30,7 +30,7 @@ var (
...
+var ERC721SafeBin = "0x608060405234801561001057600080fd5b5061010c806100206000396000f3fe6080604052348015600f57600080fd5b506004361060285760003560e01c80637354298014602d575b600080fd5b606060048036036060811015604157600080fd5b506001600160a01b038135811691602081013590911690604001356062565b005b604080516323b872dd60e01b81526001600160a01b03848116600483015230602483015260448201849052915185928316916323b872dd91606480830192600092919082900301818387803b15801560b957600080fd5b505af115801560cc573d6000803e3d6000fd5b505050505050505056fea26469706673582212202009000ef5c5ae230aaee5ae51281db209861ba168c950803d7e683c470e02ab64736f6c634300060c0033"
```
Comment (KaiserKarel) [inline bindings/ERC721Safe/ERC721Safe.go:33]: How was this derived?

Comment (frankli-dev) [reply bindings/ERC721Safe/ERC721Safe.go:33]: These are compiled bytecodes.

File: bindings/ERC20Safe/ERC20Safe.go (hunk 1 above)
Comment (frankli-dev) [reply bindings/ERC20Safe/ERC20Safe.go:33]: These are compiled bytecodes.

File: bindings/ERC721Safe/ERC721Safe.go (hunk 4 above)
Comment (KaiserKarel) [reply bindings/ERC721Safe/ERC721Safe.go:33]: Yeah, but how were they compiled? And why were they changed?

Comment (frankli-dev) [reply bindings/ERC721Safe/ERC721Safe.go:33]: @KaiserKarel Here is the command to generate go binding files.
```
TARGET=build ./scripts/setup_contracts.sh
```

**Comment by frankli-dev** [comment]
Ready to merge!

**Pull request merged**